docker/worker/Dockerfile_arm

```diff
@@ -0,0 +1,31 @@
+#!BuildTag: openqa_worker_arm
```

Comment:
I suggest to drop the new files for now and just get the existing file working on x86-64.

Also, if OBS parsing can't handle architecture-specific packages, we can drop it and re-introduce a working solution with support for multiple architectures later.